src/content/developers/docs/consensus-protocols/pos/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Proof of stake (PoS)
+description: An explanation of the proof of stake consensus protocol and its role in Ethereum.
+lang: en
+sidebar: true
+incomplete: true
+---
+
+## Prerequisites {#prerequisites}
+
+To better understand this page, we recommend you first read up on [consensus mechanisms](/developers/docs/consensus-protocols/).
+
+## What is Proof of Stake (PoS)?
+
+Proof of stake is a type of [consensus mechanism](/developers/docs/consensus-protocols/) used by blockchain networks to achieve distributed consensus. For Ethereum, the purpose of Proof of Stake is the same as [Proof of Work](/developers/docs/consensus-protocols/pow/) - to ensure that all nodes agree the state of the network and on the order of transactions.
+
+Proof of Stake comes with a number of improvements compared to [Proof of Work](/developers/docs/consensus-protocols/pow/), including: energy efficiency, lower barriers to entry, stronger crypto-economic incentives.
```

Comment:
hyphenate and I'm not sure we need to capitalise Stake and Work in these instances.